aurelia-slickgrid/src/aurelia-slickgrid/editors/singleSelectEditor.ts

```diff
@@ -119,7 +119,8 @@ export class SingleSelectEditor implements Editor {
 
   loadValue(item: any): void {
     // convert to string because that is how the DOM will return these values
-    this.defaultValue = item[this.columnDef.field].toString();
+    // make sure the prop exists first
+    this.defaultValue = item[this.columnDef.field] && item[this.columnDef.field].toString();
```

Comment:
oh I learned something new, I didn't know we could write it this way. I always write the long way of doing the check of the property then do a ternary operator